lib/liveValidation/operationValidator.ts

```diff
@@ -262,7 +282,10 @@ export const schemaValidateIssueToLiveValidationIssue = (
   input: SchemaValidateIssue[],
   operation: Operation,
   ctx: SchemaValidateContext,
-  output: LiveValidationIssue[]
+  output: LiveValidationIssue[],
+  info: OperationContext,
```

Comment:
seem the 'info' here isn't specific, could you rename it ?

Reply:
Renamed.